airbyte-integrations/connectors/source-retently/source_retently/schemas/feedback.json

```diff
@@ -0,0 +1,48 @@
+{
+  "type": "object",
+  "properties": {
+      "id": { "type": "string" },
+      "customerId": { "type": "string" },
+      "email": { "type": "string" },
+      "firstName": { "type": "string" },
+      "lastName": { "type": "string" },
+      "companyName": { "type": "string" },
+      "jobTitle": { "type": "string" },
+      "country": { "type": "string" },
+      "state": { "type": "string" },
+      "city": { "type": "string" },
+      "tags": {
+        "type": "array",
+        "items": { "type": "string" }
+      },
+      "customProps": {
+         "type": "array",
+         "items": { "type": "string" }
+       },
```

Comment:
According to the API docs, this is not an array of string but an array of objects

```json
        "customProps": [
            {
                "name" : "subscription",
                "type" : "string",
                "value" : "Professional"
            },
            {
                "name" : "signed_up",
                "type" : "date",
                "value" : "2020-05-22T00:00:00.000Z"
            }
        ],
```